index.rst

```diff
@@ -326,33 +326,162 @@ Analysis of unbinned data is clearly needed for pipeline data quality analyses,
+In particular, if the schema for a Butler dataset is defined in the `Science Data Model`_ (SDM) repository, those definitions can be used to create the corresponding tables.
+
+For the Rapid analysis use case, however, a different mechanism than the one used for data release processing to update the database tables is required, given that the information must be readily available after each visit is processed.
+
+Finally, the Visit and detector tables must enable the selection of data subsets used as input for further analysis (see `DMTN-220 Middleware Support for Campaign Definition and Management`_).
+
+For example, to assess the survey depth from single image requirements, `LPM-17 The LSST System Science Requirements Document`_ specifies the following data selection “r band images in good seeing < 0.7 arcsec (FWHM) on photometric nights (r-band sky brightness > 21 mag/arcsec2) close to the zenith (airmass < 1.2) and total exposure time of 30s.”
+
+The corresponding SQL query (using the `ImSim`_ schema as a reference) would translate to:
+
+.. code:: sql
+
+  SELECT visit
+  FROM Visit v
+  INNER JOIN ccdVisit c ON c.visitId=v.visit
+  WHERE v.band='r'
+  AND c.seeing < 0.7
+  AND c.skyBg > 21
+  AND v.airmass < 1.2
+  AND v.expTime = 30;
+
+
+**Telemetry summary tables**
+
+The current approach for combining visit information with telemetry data is a "join on time," which essentially involves querying two databases, the Butler registry to retrieve the visit's start and end times and the EFD to retrieve the telemetry values in that time range.
+
+The database `use cases`_ confluent page indicates the telemetry information that would be most useful in the Visit database.
+Typically the full time series from the EFD is not required and should consolidate the information (usually spread across multiple EFD topics) into fewer tables to make it easier to query and join with other tables in the Visit database.
+
+A tool for creating these "telemetry summary tables" in a relational database does not currently exist.
+The `SQR-58 The EFD transformation service`_ presents a possible solution by filtering and combining information from multiple EFD topics and aggregating values over time.
+This tool must be flexible enough to incrementally add new tables or add new columns to existing tables in the Visit database as needed.
+For the Rapid Analysis use case, however, loading past data into the new tables/columns is not required after evolving the schema.
+
+Note that Camera diagnostic metrics are currently recorded in a separate database.
+We recommend recording that in the Visit database as well.
+
+In addition, we could also replicate visit IDs and visit start/end times from the Butler registry into the EFD to enable the "join on time" with a single query in the EFD.
+
+**Exposure and narrative log tables**
+
+A system for exposure-level and human narrative input is in place through the `exposure log`_ and `narrative log`_ tools.
```

Comment:
"A system for exposure-level and human narrative input from the summit team is in place....

Reply:
just to emphasize the summit only bit and dodge future bullets